ipatests/test_integration/test_server_del.py

```diff
@@ -179,7 +180,7 @@ def test_ignore_topology_disconnect_replica1(self):
         )
 
         # reinstall the replica
-        tasks.uninstall_master(self.replica1)
+        tasks.uninstall_master(self.replica1, domain_level=1)
```

Comment:
you can use domain_level=self.domain_level instead of the constant 1

Reply:
you are right, somehow missed it